Circular import in commands.py

## 🐛 Bug

- `src/litdata/cli/commands.py` has this import: `from litdata.cli.parser import LitFormatter`
- But  `src/litdata/cli/parser.py` imports from commands.py as well: `from litdata.cli.commands import COMMAND_REGISTRY`

This throws an error when trying to run `make test`:


```
_______________________________________________________________________ ERROR collecting src/litdata/cli/commands.py _______________________________________________________________________
ImportError while importing test module '.../litData/src/litdata/cli/commands.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../.local/share/uv/python/cpython-3.11.14-macos-aarch64-none/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
src/litdata/cli/commands.py:18: in <module>
    from litdata.cli.parser import LitFormatter
src/litdata/cli/parser.py:17: in <module>
    from litdata.cli.commands import COMMAND_REGISTRY
E   ImportError: cannot import name 'COMMAND_REGISTRY' from partially initialized module 'litdata.cli.commands' (most likely due to a circular import) (.../litData/src/litdata/cli/commands.py)
```

### To Reproduce

Run `make setup` then `make test` on a fresh copy of the repository.

### Expected behavior

`make test` should run cleanly.

### Additional context

<!-- Add any other context about the problem here. -->

<details>
  <summary>Environment detail</summary>

- PyTorch Version (e.g., 1.0): N/A
- OS (e.g., Linux): macOS Tahoe 26.4
- How you installed PyTorch (`conda`, `pip`, source): N/A
- Build command you used (if compiling from source): `make test`
- Python version: Python 3.11
- CUDA/cuDNN version: N/A
- GPU models and configuration: N/A
- Any other relevant information: default uv `uv 0.9.26`

</details>

--- a/src/litdata/cli/commands.py
+++ b/src/litdata/cli/commands.py
@@ -13,9 +13,9 @@
 
 from argparse import _SubParsersAction
 
+from litdata.cli import LitFormatter
 from litdata.cli.handler.cache import clear_cache, show_cache_path
 from litdata.cli.handler.optimize import optimize_dataset
-from litdata.cli.parser import LitFormatter
 
 
 def register_cache_subcommand(subparser: _SubParsersAction) -> None:

